app/views/concepts/_form.html.erb

```diff
@@ -12,10 +12,10 @@
   <% end %>
 
   <div class="field">
-    <%= f.label :name %><br>
-    <%= f.text_field :name %>
+    <%# f.label :name %>
```

Comment:
Here's an article that explains more of what Drew is referencing: http://stackoverflow.com/questions/19758598/what-is-sr-only-in-bootstrap-3